lib/zendesk_apps_support/product.rb

```diff
@@ -19,11 +19,13 @@ def self.all
     PRODUCTS_AVAILABLE = [
       Product.new(code: 1, name: 'support', legacy_name: 'zendesk'),
       Product.new(code: 2, name: 'chat', legacy_name: 'zopim'),
-      Product.new(code: 3, name: 'standalone_chat', legacy_name: 'lotus_box')
+      Product.new(code: 3, name: 'standalone_chat', legacy_name: 'lotus_box'),
+      Product.new(code: 4, name: 'sell', legacy_name: 'base')
```

Comment:
@anyone Do I need to populate these in the DB? If yes, how can I do it?

Reply:
Also, not sure If Sell needs `legacy_name`? We did not appear under legacy name in Zendesk Suite...

Reply:
@grzegorz-dyk I see `legacy_name` used here: https://github.com/zendesk/zendesk_app_market/blob/9036852175a27ddb40a1d97d9b8861951053df18/app/models/location.rb#L19

we could use the .name if .legacy_name doesn't exist in `app_market` or simply define `sell` in both cases.